### PR TITLE
Update visu_config

### PR DESCRIPTION
**Pull request by SirTobiIV**
add attribute explanation to widget page, navbar

### CODE REVIEW

**Comment by SirTobiIV** [comment]
GIT is shit! such a crappy complictaed thing. I think I now got it how to transfer the local changes on my computer to github.

Is it possible that I can only have one pull request at a time? Or is that a restriction of my windows git client?

**Comment by ChristianMayer** [comment]
You can have as many as you want - but it's one per branch.

So you should create a branch of your develop, modify it and then have a pull request from that. (Once it's pulled you can delete it)
Then just start the next branch with the next pull request.

File: src/visu_config.xsd [hunk 1]
```diff
@@ -598,12 +598,42 @@
...
     </xsd:attribute>
-    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-top" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-bottom" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-left" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-right" type="xsd:boolean" use="optional" />
+    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">the navigation path shows on which subpage of the visu one currently is</xsd:documentation>
```
Comment (ChristianMayer) [inline src/visu_config.xsd]: The sentences are usually starting with a capital letter. And this is quite informal wording.

I'd suggest something like: Select the display of the top navigation that shows the currently displayed page with all parent pages.

File: src/visu_config.xsd [hunk 2]
```diff
@@ -598,12 +598,42 @@
...
+            <xsd:documentation xml:lang="en">Der Navigationspfad zeigt an, auf welcher Seite der Visualisierung man sich befindet</xsd:documentation>
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the CometVisu logo and various Links</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt die Fußzeile mit CometVisu Logo und diversen Links an</xsd:documentation>
```
Comment (ChristianMayer) [inline src/visu_config.xsd]: The CometVisu logo and the links are just one way to use the footer. The content is set in the config and thus can show many different this

File: src/visu_config.xsd [hunk 3]
```diff
@@ -632,8 +667,18 @@
...
-    <xsd:attribute name="position" type="navbarPositionType" use="required" />
+    <xsd:attribute name="name" type="xsd:string" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Name/title of the navigation bar</xsd:documentation>
+            <xsd:documentation xml:lang="en">Name/Überschrift der Navigationsleiste</xsd:documentation>
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="position" type="navbarPositionType" use="required">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">position of the navigation bar on the page</xsd:documentation>
```
Comment (ChristianMayer) [inline src/visu_config.xsd]: Please start with a capital letter

File: src/visu_config.xsd [hunk 4]
```diff
@@ -598,12 +598,42 @@
...
-    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-top" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-bottom" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-left" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-right" type="xsd:boolean" use="optional" />
+    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">the navigation path shows on which subpage of the visu one currently is</xsd:documentation>
+            <xsd:documentation xml:lang="en">Der Navigationspfad zeigt an, auf welcher Seite der Visualisierung man sich befindet</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd (hunk 2 above)
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 5]
```diff
@@ -598,12 +598,42 @@
...
+    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the CometVisu logo and various Links</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt die Fußzeile mit CometVisu Logo und diversen Links an</xsd:documentation>
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shownavbar-top" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the defined navigation bar on top of the page</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt oben auf der Seite die definierte Navigationsleiste an</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 6]
```diff
@@ -598,12 +598,42 @@
...
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shownavbar-bottom" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the defined navigation bar at the bottom of the page</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt unten auf der Seite die definierte Navigationsleiste an</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 7]
```diff
@@ -598,12 +598,42 @@
...
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shownavbar-left" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the defined navigation bar at the left side of the page</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt links auf der Seite die definierte Navigationsleiste an</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 8]
```diff
@@ -598,12 +598,42 @@
...
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="shownavbar-right" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Shows the defined navigation bar at the right side of the page</xsd:documentation>
+            <xsd:documentation xml:lang="en">Zeigt rechts auf der Seite die definierte Navigationsleiste an</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 9]
```diff
@@ -617,7 +647,12 @@
         <xsd:group ref="AvailablePlugins"/>
       </xsd:choice>
     </xsd:sequence>
-    <xsd:attribute name="name" type="xsd:string" use="optional" />
+    <xsd:attribute name="name" type="xsd:string" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Name/title of the group</xsd:documentation>
+            <xsd:documentation xml:lang="en">Name/Überschrift der Gruppe</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 10]
```diff
@@ -632,8 +667,18 @@
       <xsd:group ref="Widgets"/>
       <xsd:group ref="AvailablePlugins"/>
     </xsd:choice>
-    <xsd:attribute name="name" type="xsd:string" use="optional" />
-    <xsd:attribute name="position" type="navbarPositionType" use="required" />
+    <xsd:attribute name="name" type="xsd:string" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">Name/title of the navigation bar</xsd:documentation>
+            <xsd:documentation xml:lang="en">Name/Überschrift der Navigationsleiste</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

File: src/visu_config.xsd [hunk 11]
```diff
@@ -632,8 +667,18 @@
...
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="position" type="navbarPositionType" use="required">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">position of the navigation bar on the page</xsd:documentation>
+            <xsd:documentation xml:lang="en">Position der Navigationsleiste auf der Seite</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: xml:lang="de"

**Comment by SirTobiIV** [comment]
hope it is now ok like that?

**Comment by peuter** [comment]
Yes everything is fine, thanks!

**Pull request merged**